docs/root/intro/version_history.rst

```diff
@@ -39,6 +39,7 @@ Version history
 * upstream: added runtime error checking to prevent setting dns type to STRICT_DNS or LOGICAL_DNS when custom resolver name is specified.
 * grpc-json: added support for :ref:`auto mapping
   <envoy_api_field_config.filter.http.transcoder.v2.GrpcJsonTranscoder.auto_mapping>`.
+* access log: added a new field for downstream TLS session ID to file and gRPC access logger.
```

Comment:
Please put your entry near the top (so that it's alphabetical order), as it helps cut down on merge conflicts in this file. Also, would you mind moving the grpc-json entry as well?

Reply:
Done